.circleci/config.yml

```diff
@@ -6,6 +6,24 @@ orbs:
   aws-ecr: circleci/aws-ecr@9 # this orb doesn't support OIDC v2, so we use aws-cli to authenticate
 
 
+commands:
+  aws-cli-setup-and-tagging:
```

Comment:
I'm unsure about this name. It looks like it logs into aws ECR - should it not be called aws-login-to-ecr or something similar?

Reply:
Thanks, Stephen. The typical problem of naming something. I agree, that's a better name.

Reply:
What about `aws-login-to-ecr-and-tagging` I put tagging as we create target tags as well in the command